Python/Product/Django/Project/DjangoPropertyPage.cs

```diff
@@ -30,6 +30,16 @@ class DjangoPropertyPage : CommonPropertyPage {
         public DjangoPropertyPage() {
             _control = new DjangoPropertyPageControl(this);
         }
+
```

Comment:
Same thing here, who's calling this?  Also why are we introducing protected members into classes with no subclasses?  Neither of these will ever be called with Dispose(false)